cdap-ui/graphql/resolvers-common.js

```diff
@@ -41,14 +41,24 @@ export function requestPromiseWrapper(options, token, bodyModifiersFn) {
   return new Promise((resolve, reject) => {
     request(options, (err, response, body) => {
       if (err) {
-        const error =  new ApolloError(err, '500');
-        return reject(error);
+        let exception;
+        if (typeof errorModifiersFn === 'function') {
+          exception = errorModifiersFn(err, '500');
```

Comment:
Don't think we should hardcode 500 here. We might have 403 or 401 errors tomorrow which should not converted. Can we get the status code from the request response?

Reply:
done